src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -476,86 +596,14 @@ def get_chemical_symbols(self, logger: 'BoundLogger') -> list[str]:
-
-        Returns:
-            (Optional[ase.Atoms]): The ASE Atoms object with the basic information from the `AtomicCell`.
-        """
-        # Initialize ase.Atoms object with labels
-        atoms_labels = self.get_chemical_symbols(logger=logger)
-        ase_atoms = ase.Atoms(symbols=atoms_labels)
-
-        # PBC
-        if self.periodic_boundary_conditions is None:
-            logger.info(
-                'Could not find `AtomicCell.periodic_boundary_conditions`. They will be set to [False, False, False].'
-            )
-            self.periodic_boundary_conditions = [False, False, False]
-        ase_atoms.set_pbc(pbc=self.periodic_boundary_conditions)
-
-        # Lattice vectors
-        if self.lattice_vectors is not None:
-            ase_atoms.set_cell(cell=self.lattice_vectors.to('angstrom').magnitude)
-        else:
-            logger.info('Could not find `AtomicCell.lattice_vectors`.')
-
-        # Positions
-        if self.positions is not None:
-            if len(self.positions) != len(self.atoms_state):
-                logger.error(
-                    'Length of `AtomicCell.positions` does not coincide with the length of the `AtomicCell.atoms_state`.'
-                )
-                return None
-            ase_atoms.set_positions(
-                newpositions=self.positions.to('angstrom').magnitude
-            )
-        else:
-            logger.warning('Could not find `AtomicCell.positions`.')
-            return None
-
-        return ase_atoms
-
-    def from_ase_atoms(self, ase_atoms: ase.Atoms, logger: 'BoundLogger') -> None:
-        """
-        Parses the information from an ASE Atoms object to the `AtomicCell` section.
-
-        Args:
-            ase_atoms (ase.Atoms): The ASE Atoms object to parse.
-            logger (BoundLogger): The logger to log messages.
-        """
-        # `AtomsState[*].chemical_symbol`
-        for symbol in ase_atoms.get_chemical_symbols():
-            atom_state = AtomsState(chemical_symbol=symbol)
-            self.atoms_state.append(atom_state)
-
-        # `periodic_boundary_conditions`
-        self.periodic_boundary_conditions = ase_atoms.get_pbc()
-
-        # `lattice_vectors`
-        cell = ase_atoms.get_cell()
-        self.lattice_vectors = ase.geometry.complete_cell(cell) * ureg('angstrom')
-
-        # `positions`
-        positions = ase_atoms.get_positions()
-        if (
-            not positions.tolist()
-        ):  # ASE assigns a shape=(0, 3) array if no positions are found
-            return None
-        self.positions = positions * ureg('angstrom')
-
     def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         super().normalize(archive, logger)
 
         # Set the name of the section
         self.name = self.m_def.name if self.name is None else self.name
 
 
+# TODO Consider changing name to BeadCell or CGBeadCell, only using "particle" for the more abstract reference to atoms or beads
```

Comment:
@Bernadette-Mohr how would you feel about this type of naming?

Reply:
That makes a lot of sense!
Distinctive, clear naming goes a long way toward well-structured, understandable code.